programs/nft-voter/src/state/max_voter_weight_record.rs

```diff
@@ -0,0 +1,69 @@
+use crate::id;
+use anchor_lang::prelude::Pubkey;
+use anchor_lang::prelude::*;
+
+/// MaxVoterWeightRecord account as defined in spl-governance-addin-api
+/// It's redefined here without account_discriminator for Anchor to treat it as native account
```

Comment:
It might be worth creating an Anchor-specific `addin-api` crate to avoid having to do this everywhere

Reply:
No, it's not possible at the moment because `Anchor` needs the accounts inside the create. It wraps them into a struct with the current `program_id`. 
Also if they are not in the same crate then it won't generate `IDL` fort hem.
The only option would be to use a macro generating the struct for the current crate, but not sure if it's any better.

Reply:
Ah, that's too bad, makes sense